__tests__/app.test.js

```diff
@@ -155,4 +155,71 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("POST /api/articles/:article_id/comments ", () => {
+        test("201: should returned and update/post body comments table body column with article_id", () => {
+            return request(app)
+                .post("/api/articles/9/comments")
+                .expect(201)
+                .send({
+                    username: "butter_bridge",
+                    body: "post request will going to be ok",
+                })
+                .then(({ body }) => {
+                    const { comment } = body;
+                    expect(comment).toBe("post request will going to be ok");
```

Comment:
we want the comment to come back with more information than just the body :) what about comment_id , article_id etc